karton/classifier/classifier.py

```diff
@@ -259,6 +259,45 @@ def zip_has_file(path: str) -> bool:
             sample_class.update({"kind": "runnable", "platform": "linux"})
             return sample_class
 
+        # Is PKG file?
+        if magic.startswith("xar archive") or extension == "pkg":
+            sample_class.update(
+                {"kind": "runnable", "platform": "macos", "extension": "pkg"}
+            )
+            return sample_class
+
+        # Is DMG file?
+        if (
+                extension == "dmg" or
+                all([len(content) > 512,
+                     content[-512:][:4] == b"koly",
+                     content[-512:][8:12] == b"\x00\x00\x02\x00"])
+        ):
+            sample_class.update(
+                {"kind": "runnable", "platform": "macos", "extension": "dmg"}
+            )
+            return sample_class
+
+        # Is mach-o file?
+        if magic.startswith("Mach-O"):
+            sample_class.update(
+                {"kind": "runnable", "platform": "macos"}
+            )
+            return sample_class
+
+        def zip_has_mac_app() -> bool:
+            zipfile = ZipFile(BytesIO(content))
+            if any(x.filename.lower().endswith(".app/contents/info.plist") for x in zipfile.filelist):
+                return True
+            return False
```

Comment:
I think this can be simplified to:

```suggestion
            return any(x.filename.lower().endswith(".app/contents/info.plist") for x in zipfile.filelist):
```
?